server/processor.go

```diff
@@ -36,14 +35,11 @@ type PostProcessor struct {
 	botID     string
 	moderator moderation.Moderator
 
-	stopChan chan bool
-
 	thresholdValue   int
 	excludedUsers    map[string]struct{}
 	excludedChannels map[string]struct{}
 
-	postsToProcess []*model.Post
-	processLock    sync.Mutex
+	posts chan *model.Post
```

Comment:
Can we name this something to show it is not an array of Posts, but is a channel?